src/System.Globalization/tests/CultureInfo/CultureInfoCurrentCulture.cs

```diff
@@ -57,6 +57,7 @@ public void CurrentUICulture()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Thread cultures is not honored in UWP.")]
```

Comment:
No this is by design. in app container the default culture is what WinRT uses and not the the culture attached to the thread. it is just how the app model designed for WinRT